test/plugin_functional/plugins/kbn_tp_custom_visualizations/public/self_changing_vis/self_changing_vis.js

```diff
@@ -1,47 +0,0 @@
-/*
- * Licensed to Elasticsearch B.V. under one or more contributor
- * license agreements. See the NOTICE file distributed with
- * this work for additional information regarding copyright
- * ownership. Elasticsearch B.V. licenses this file to you under
- * the Apache License, Version 2.0 (the "License"); you may
- * not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *    http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing,
- * software distributed under the License is distributed on an
- * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
- * KIND, either express or implied.  See the License for the
- * specific language governing permissions and limitations
- * under the License.
- */
-
-import { SelfChangingEditor } from './self_changing_editor';
-import { SelfChangingComponent } from './self_changing_components';
-
-import { npSetup } from '../../../../../../src/legacy/ui/public/new_platform';
-
-npSetup.plugins.visualizations.createReactVisualization({
-  name: 'self_changing_vis',
-  title: 'Self Changing Vis',
```

Comment:
Remains from before this test plugin was migrated to KP. The file was unused (the test component is now registered from `test/plugin_functional/plugins/kbn_tp_custom_visualizations/public/plugin.ts`)